kong/clustering/compat/checkers.lua

```diff
@@ -23,6 +23,70 @@ end
 
 
 local compatible_checkers = {
+  { 3005000000, --[[ 3.5.0.0 ]]
+    function(config_table, dp_version, log_suffix)
+      local has_update
+
+      for _, plugin in ipairs(config_table.plugins or {}) do
+        if plugin.name == 'opentelemetry' or plugin.name == 'zipkin' then
+          local config = plugin.config
+          if config.header_type == 'gcp' then
+            config.header_type = 'preserve'
+            log_warn_message('contains configuration header_type == gcp',
+                              'overwritten with default value `preserve`',
+                              dp_version, log_suffix)
```

Comment:
Could we include some more context in these warning messages? Maybe at least the name of the plugin?